R/utils.R

```diff
@@ -349,27 +349,30 @@ chunk_inputs <- function(chunks = find_chunks()$name) {
   assertthat::assert_that(is.character(chunks))
 
   # Get list of data required by each chunk
-  chunkinputs <- list()
+  names <- character()
```

Comment:
This masks `names`, a very common base R function--would be better to use something else.